provider/win_driver_utils.py

```diff
@@ -204,20 +316,13 @@ def memory_leak_check(vm, test, params):
     :param test: kvm test object
     :param params: the dict used for parameters
     """
-    driver_name = params["driver_name"]
-    devcon_path = params["devcon_path"]
-    media_type = params.get("virtio_win_media_type", "iso")
-
-    device_name = driver_info_dict[driver_name]["device_name"]
-    device_hwid = driver_info_dict[driver_name]["hwid"]
     session = vm.wait_for_login()
-    uninstall_driver(session, test, devcon_path, driver_name,
-                     device_name, device_hwid)
+    unload_driver(vm, test, params, load_method)
     time.sleep(10)
     if vm.is_alive() is False:
         test.fail("VM is not alive after uninstall driver,"
                   "please check if it is a memory leak")
-    session = vm.reboot(session)
-    install_driver_by_virtio_media(session, test, devcon_path,
-                                   media_type, driver_name, device_hwid)
+    if load_method != 'enable':
+        session = vm.reboot(session)
```

Comment:
Seems we can use `vm.reboot()` directly,and remove line319(`session = vm.wait_for_login()`).Because we do not need `session` in this function now.
Hi @vivianQizhu What do you think about it?

Reply:
@leidwang Yes, you are right.